app/components/primer/tooltip_component.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+
+module Primer
+  # The Tooltip component is a wrapper component that will apply a tooltip to the provided content.
+  class TooltipComponent < Primer::Component
+    DIRECTION_DEFAULT = :n
+    ALIGN_DEFAULT = :default
+    MULTILINE_DEFAULT = false
+    DELAY_DEFAULT = false
+
+    ALIGN_MAPPING = {
+      ALIGN_DEFAULT => "",
+      :left_1 => "tooltipped-align-left-1",
+      :right_1 => "tooltipped-align-right-1",
+      :left_2 => "tooltipped-align-left-2",
+      :right_2 => "tooltipped-align-right-2",
+    }
+
+    DIRECTION_OPTIONS = [DIRECTION_DEFAULT] + [
+      :nw,
+      :ne,
+      :w,
+      :e,
+      :sw,
+      :s,
+      :se,
+    ]
+
+    # @example 100|Default
+    #   <div class="pt-5">
+    #     <%= render(Primer::TooltipComponent.new(label: "Even bolder")) { "Default Bold Text" } %>
+    #   </div>
+    #
+    # @example 100|Wrapping another component
+    #   <div class="pt-5">
+    #     <%= render(Primer::TooltipComponent.new(label: "Even bolder")) do %>
+    #       <%= render(Primer::ButtonComponent.new) { "Bold Button" } %>
+    #     <% end %>
+    #   </div>
+    #
+    # @example 100|With a direction
+    #   <div class="pt-5">
+    #     <%= render(Primer::TooltipComponent.new(label: "Even bolder", direction: :s)) { "Bold Text With a Direction" } %>
+    #   </div>
+    #
+    # @example 100|With an alignment
+    #   <div class="pt-5">
+    #     <%= render(Primer::TooltipComponent.new(label: "Even bolder", direction: :s, alignment: :right_1)) { "Bold Text With an Alignment" } %>
+    #   </div>
+    #
+    # @example 100|Without a delay
+    #   <div class="pt-5">
+    #     <%= render(Primer::TooltipComponent.new(label: "Even bolder", direction: :s, no_delay: true)) { "Bold Text without a delay" } %>
+    #   </div>
```

Comment:
@srt32 as a quick follow-up, might we pass `pt: 5` to these components instead of having `pt-5` the wrapping div?

Reply:
With that change, we still can't see the tooltip with the iframe as is.

```
     # @example 100|Default
-    #   <div class="pt-5">
-    #     <%= render(Primer::TooltipComponent.new(label: "Even bolder")) { "Default Bold Text" } %>
-    #   </div>
+    #   <%= render(Primer::TooltipComponent.new(label: "Even bolder", pt: 5)) { "Default Bold Text" } %>
     #
```


![image](https://user-images.githubusercontent.com/2181356/106642753-ff947400-6545-11eb-8830-9af48f91b740.png)

Am I missing something there?

Reply:
maybe adding `mt-5` or even both `mt-5` and `pt-5` may solve it?